kong/cmd/utils/prefix_handler.lua

```diff
@@ -103,6 +103,24 @@ local function gen_default_ssl_cert(kong_config, target)
   return true
 end
 
+
+local function gen_trusted_certs_combined_file(
+  combined_filepath,
+  lua_ssl_trusted_certificate_combined)
+
+  log.verbose("generating trusted certs combined file in ",
+              combined_filepath)
+
+  local fd = assert(io.open(combined_filepath, "w"))
+
+  for _, path in ipairs(lua_ssl_trusted_certificate_combined) do
+    fd:write(pl_file.read(path))
```

Comment:
Since we are appending the certificate, i'd suggest to add a `\n` for each file, in case some one incorrectly specifies a file without newline.